### PR TITLE
Update source schema export page

### PR DESCRIPTION
**Pull request by ranand**
### Proposed changes
- We worked on updating the source schema export page to include:
  - properties with Track event exports
  - Download CSV capability for Identify & Group
  - Epic: https://segment.atlassian.net/browse/PROT-3830

### Merge timing
- 11/22/2022 (Tuesday)

### CODE REVIEW

File: src/connections/destination-data-control.md [hunk 1]
```diff
@@ -42,14 +42,19 @@ Once you block an event in Segment, Segment stops forwarding it to all of your D
...
 ![](images/export-source-schema.png)
 
+You can choose to include properties with the Track events in the modal.
+
+![](images/export-source-schema-including-props.png)
+
 CSV files generate based on the current view of your Source Schema. Any search parameters or filters you apply to the current Source Schema view also apply to the CSV.
 
-You can download the following Track events:
+The Track event CSV file will contain the following columns:
```
Comment (markzegarelli) [inline src/connections/destination-data-control.md:55]:
```suggestion
The Track event CSV file contains the following columns:
```

File: src/connections/destination-data-control.md [hunk 2]
```diff
@@ -42,14 +42,19 @@ Once you block an event in Segment, Segment stops forwarding it to all of your D
 
 ## Export your Source Schema
 
-Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the Source Schema page.
+Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the Source Schema page. This is supported for Track, Identify, and Group events.
```
Comment (markzegarelli) [inline src/connections/destination-data-control.md:45]:
```suggestion
Segment allows you to download your Source Schema as a CSV file, maximizing portability and access to event data. You can download a copy of your schema by visiting the Source Schema page. This is supported for Track, Identify, and Group events. Track, Identify, and Group events support Source Schema export.
```

File: src/connections/destination-data-control.md [hunk 3]
```diff
@@ -58,7 +63,15 @@ You can download the following Track events:
 - Planned (available for Protocols customers with a connected Tracking Plan)
   - Values are "planned" or "unplanned"
 
-Export Schema is not available for Identify or Group events.
+The Identify & Group CSV file will contain the following columns:
```
Comment (markzegarelli) [inline src/connections/destination-data-control.md:66]:
```suggestion
The Identify & Group CSV file contains the following columns:
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/destination-data-control.md (hunk 1 above)
Comment (ranand) [reply src/connections/destination-data-control.md]: done!

File: src/connections/destination-data-control.md (hunk 2 above)
Comment (ranand) [reply src/connections/destination-data-control.md]: done!

File: src/connections/destination-data-control.md (hunk 3 above)
Comment (ranand) [reply src/connections/destination-data-control.md]: done!

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
Will merge this on Friday to ensure it makes it on 11/22

**Pull request merged**